src/chart/heatmap/HeatmapView.ts

```diff
@@ -260,30 +265,39 @@ class HeatmapView extends ChartView {
                 if (isNaN(data.get(dataDims[1], idx) as number)) {
                     continue;
                 }
+                const shape: RectShape = coordSys.dataToRect([data.get(dataDims[0], idx)]).contentShape;
```

Comment:
This change of creating a temporary variable `shape` can be undo. So there is no necessity of using type `RectShape`.

Reply:
I've fix it.